securedrop/tests/conftest.py

```diff
@@ -84,42 +85,53 @@ def setUpTearDown():
     _cleanup_test_securedrop_dataroot(original_config)
 
 
+@pytest.fixture(scope="session")
+def gpg_key_dir() -> Iterator[Path]:
```

Comment:
Shouldn't this be https://docs.python.org/3/library/typing.html#typing.Generator ?